Broken Cosmos DB integration tests

# Bug Report

## Describe the Bug 
Cosmos DB CI tests are broken on `main`

https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/runs/6454242008?check_suite_focus=true

```
CosmosAssetIndexIntegrationTest > findAll_withInvalidFilter_throwsException() FAILED
    Expecting message to be:
      "Cannot build SqlParameter for operator: STARTSWITH"
    but was:
      "Cannot build WHERE clause, reason: unsupported operator STARTSWITH"


CosmosAssetIndexIntegrationTest > queryAssets_operatorIn_notFound() FAILED
Syntax error, incorrect syntax near '@asset_prop_id'

CosmosAssetIndexIntegrationTest > queryAssets_operatorIn() FAILED
Syntax error, incorrect syntax near '@asset_prop_id'

CosmosAssetIndexIntegrationTest > queryAssets_operatorIn_noUpTicks() FAILED
Syntax error, incorrect syntax near '@asset_prop_id'

CosmosAssetIndexIntegrationTest > findAll_withFilteringOperatorIn_limitExceedsResultSize() FAILED
Syntax error, incorrect syntax near '@foo'.
```

Reopening as still failing after #1315 
```

2022-05-16T18:59:46.4965182Z CosmosContractNegotiationStoreIntegrationTest > findAll_verifyFiltering_invalidFilterExpression() FAILED
2022-05-16T18:59:46.4977296Z     org.opentest4j.AssertionFailedError: 
2022-05-16T18:59:46.4977729Z     Expecting message to be:
2022-05-16T18:59:46.4978179Z       "Cannot build SqlParameter for operator: contains"
2022-05-16T18:59:46.4978519Z     but was:
2022-05-16T18:59:46.4978886Z       "Cannot build WHERE clause, reason: unsupported operator contains"
2022-05-16T18:59:46.4979230Z 


2022-05-16T19:01:38.0964483Z 
2022-05-16T19:01:38.0965160Z CosmosContractNegotiationStoreIntegrationTest > findAll_verifySortingInvalidSortField() FAILED
2022-05-16T19:01:38.0965689Z     java.lang.AssertionError: 
2022-05-16T19:01:38.0966343Z     Expecting empty but was: [org.eclipse.dataspaceconnector.spi.types.domain.contract.negotiation.ContractNegotiation@28e66e21,



2022-05-16T19:03:04.9965006Z CosmosContractNegotiationStoreIntegrationTest > queryAgreements_verifySorting_invalidProperty() FAILED
2022-05-16T19:03:04.9965900Z     java.lang.AssertionError: 
2022-05-16T19:03:04.9966624Z     Expecting empty but was: [org.eclipse.dataspaceconnector.spi.types.domain.contract.agreement.ContractAgreement@2b49c824,
2022-05-16T19:03:04.9967458Z         org.eclipse.dataspaceconnector.spi.types.domain.contract.agreement.ContractAgreement@fb1e82be,
2022-05-16T19:03:05.0034560Z         org.eclipse.dataspaceconnector.spi.types.domain.contract.agreement.ContractAgreement@ec5d7310,
2022-05-16T19:03:05.0035400Z         org.eclipse.dataspaceconnector.spi.types.domain.contract.agreement.ContractAgreement@f8191a16,
2022-05-16T19:03:05.0037329Z         org.eclipse.dataspaceconnector.spi.types.domain.contract.agreement.ContractAgreement@3588c1f5,

--- a/extensions/azure/cosmos/contract-negotiation-store-cosmos/src/test/java/org/eclipse/dataspaceconnector/contract/negotiation/store/CosmosContractNegotiationStoreIntegrationTest.java
+++ b/extensions/azure/cosmos/contract-negotiation-store-cosmos/src/test/java/org/eclipse/dataspaceconnector/contract/negotiation/store/CosmosContractNegotiationStoreIntegrationTest.java
@@ -305,7 +305,7 @@ class CosmosContractNegotiationStoreIntegrationTest {
         var state = ContractNegotiationStates.CONFIRMED;
         var n = generateNegotiation(state);
         var doc = new ContractNegotiationDocument(n, partitionKey);
-        Duration leaseDuration = Duration.ofSeconds(2);
+        Duration leaseDuration = Duration.ofSeconds(10); // give it some time to compensate for TOF delays
         doc.acquireLease("another-connector", leaseDuration);
         container.createItem(doc);
 
@@ -314,7 +314,7 @@ class CosmosContractNegotiationStoreIntegrationTest {
         assertThat(negotiationsBeforeLeaseExpired).isEmpty();
         // after the lease expired
         await()
-                .atMost(Duration.ofSeconds(10))
+                .atMost(Duration.ofSeconds(20))
                 .pollInterval(Duration.ofMillis(500))
                 .pollDelay(leaseDuration) //give the lease time to expire
                 .untilAsserted(() -> {
@@ -420,7 +420,7 @@ class CosmosContractNegotiationStoreIntegrationTest {
 
         var query = QuerySpec.Builder.newInstance().filter("something contains other").build();
 
-        assertThatThrownBy(() -> store.queryNegotiations(query)).isInstanceOfAny(IllegalArgumentException.class).hasMessage("Cannot build SqlParameter for operator: contains");
+        assertThatThrownBy(() -> store.queryNegotiations(query)).isInstanceOfAny(IllegalArgumentException.class).hasMessage("Cannot build WHERE clause, reason: unsupported operator contains");
     }
 
     @Test
@@ -447,7 +447,7 @@ class CosmosContractNegotiationStoreIntegrationTest {
         IntStream.range(0, 10).mapToObj(i -> generateDocument()).forEach(d -> container.createItem(d));
 
         var query = QuerySpec.Builder.newInstance().sortField("xyz").sortOrder(SortOrder.ASC).build();
-        assertThat(store.queryNegotiations(query)).isEmpty();
+        assertThat(store.queryNegotiations(query)).hasSize(10);
     }
 
     @Test
@@ -538,7 +538,7 @@ class CosmosContractNegotiationStoreIntegrationTest {
 
         var query = QuerySpec.Builder.newInstance().sortField("notexist").sortOrder(SortOrder.DESC).build();
 
-        assertThat(store.queryAgreements(query)).isEmpty();
+        assertThat(store.queryAgreements(query)).hasSize(10);
     }
 
     private ContractNegotiationDocument toDocument(Object object) {

--- a/extensions/azure/cosmos/transfer-process-store-cosmos/src/test/java/org/eclipse/dataspaceconnector/transfer/store/cosmos/CosmosTransferProcessStoreIntegrationTest.java
+++ b/extensions/azure/cosmos/transfer-process-store-cosmos/src/test/java/org/eclipse/dataspaceconnector/transfer/store/cosmos/CosmosTransferProcessStoreIntegrationTest.java
@@ -169,7 +169,7 @@ class CosmosTransferProcessStoreIntegrationTest {
         String id1 = UUID.randomUUID().toString();
         var tp = createTransferProcess(id1, TransferProcessStates.INITIAL);
         TransferProcessDocument item = new TransferProcessDocument(tp, partitionKey);
-        Duration leaseDuration = Duration.ofSeconds(2);
+        Duration leaseDuration = Duration.ofSeconds(10);
         item.acquireLease("another-connector", leaseDuration);
         container.upsertItem(item);
 
@@ -177,7 +177,7 @@ class CosmosTransferProcessStoreIntegrationTest {
         assertThat(processesBeforeLeaseBreak).isEmpty();
 
         await()
-                .atMost(Duration.ofSeconds(10))
+                .atMost(Duration.ofSeconds(20))
                 .pollInterval(Duration.ofMillis(500))
                 .pollDelay(leaseDuration) //give the lease time to expire
                 .untilAsserted(() -> {
@@ -564,7 +564,7 @@ class CosmosTransferProcessStoreIntegrationTest {
 
         var query = QuerySpec.Builder.newInstance().filter("something contains other").build();
 
-        assertThatThrownBy(() -> store.findAll(query)).isInstanceOfAny(IllegalArgumentException.class).hasMessage("Cannot build SqlParameter for operator: contains");
+        assertThatThrownBy(() -> store.findAll(query)).isInstanceOfAny(IllegalArgumentException.class).hasMessage("Cannot build WHERE clause, reason: unsupported operator contains");
     }
 
     @Test

